data/members.json

```diff
@@ -0,0 +1,50 @@
+  { "name":"Dart Ken Poon", "title": "Aerodynamics", "location": "", "imageUrl": "assets/images/members/dart_ken_poon.png" },
+  { "name":"Corey Stein", "title": "Aerodynamics", "location": "", "imageUrl": "assets/images/members/corey_stein.png" },
+  { "name":"Peter Stephan Gschladt", "title": "Systems", "location": "Germany", "imageUrl": "assets/images/members/peter_stephan_gschladt.png" },
+  { "name":"Kevin Miller", "title": "Aerodynamics/Mechanical", "location": "USA", "imageUrl": "assets/images/members/kevin_miller.png" },
+  { "name":"Clarence Akagu", "title": "Mechanical", "location": "Nigeria", "imageUrl": "assets/images/members/clarence_akagu.png" },
+  { "name":"Michael Elmore", "title": "Numbers Simulation", "location": "USA", "imageUrl": "assets/images/members/michael_elmore.png" },
+  { "name":"Jenico J", "title": "Numbers Simulation", "location": "India", "imageUrl": "assets/images/members/jenico_j.png" },
+  { "name":"Lachlan Grogan", "title": "Systems", "location": "Australia", "imageUrl": "assets/images/members/lachlan_grogan.png" },
+  { "name":"Alan Espinoza", "title": "Engineering Systems", "location": "Nigeria", "imageUrl": "assets/images/members/alan_espinoza.png" },
+  { "name":"Corne Kooyman", "title": "Web Development", "location": "South Africa", "imageUrl": "assets/images/members/corne_kooyman.png" },
+  { "name":"Jari van der Salm", "title": "Finance", "location": "Holland", "imageUrl": "assets/images/members/jari_van_der_salm.png" },
+  { "name":"Gopal Bidari", "title": "Video", "location": "USA", "imageUrl": "assets/images/members/gopal_bidari.png" },
+  { "name":"Cody Huang", "title": "Electrical", "location": "USA", "imageUrl": "assets/images/members/cody_huang.png" },
+  { "name":"Denis Tudor", "title": "Electrical", "location": "Romania", "imageUrl": "assets/images/members/denis_tudor.png" },
+  { "name":"Cihan Bebek", "title": "Web Development", "location": "Finland", "imageUrl": "assets/images/members/cihan_bebek.png" },
+  { "name":"Matthew Marshall", "title": "Finance", "location": "Canada", "imageUrl": "assets/images/members/matthew_marshall.png" },
+  { "name":"Alex Hale", "title": "Public Relations", "location": "United Kingdom", "imageUrl": "assets/images/members/alex_hale.png" },
+  { "name":"Nathaniel Thompson", "title": "Public Relations", "location": "", "imageUrl": "assets/images/members/nathaniel_thompson.png" },
+  { "name":"Ralph Steenblik", "title": "Public Relations", "location": "", "imageUrl": "assets/images/members/ralph_steenblik.png" },
+  { "name":"Matt Malcom", "title": "Public Relations", "location": "New Zealand", "imageUrl": "assets/images/members/matt_malcom.png" },
+  { "name":"Joseph Cirnigliaro", "title": "Electrical/Manufacturing", "location": "USA?", "imageUrl": "assets/images/members/joseph_cirnigliaro.png" },
+  { "name":"Nick Pope", "title": "Manufacturing", "location": "USA", "imageUrl": "assets/images/members/nick_pope.png" }
+
```

Comment:
are you referring to line 49? if so I'm not sure, I don't totally know the requirements of a .json file, beyond understanding the idea of creating different values for different fields as a sort of database